src/views/user/AnonymousLogin.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useState } from "react";
+import { Modal, Form, Image, Header, Message } from "semantic-ui-react";
+import { UserContext } from "contexts/UserContext";
+import { useTranslation } from "react-i18next";
+
+const AnonymousLoginPage = () => {
+  const [nickname, setNickname] = useState("");
+  const [error, setError] = useState(null);
+  const { t } = useTranslation();
+
+  const { logInAnonymously } = useContext(UserContext);
+
+  const handleLogin = async () => {
+    try {
+      await logInAnonymously(nickname);
+    } catch (e) {
+      setError(e.message);
+    }
+  };
+
+  return (
+    <Modal open={true}>
+      <Modal.Header>{t("login.title")}</Modal.Header>
+      <Modal.Content image>
+        <Image wrapped size="medium" src="/images/game-hugo.png" />
+        <Modal.Description>
+          <Header>{t("login.actions.guest_login")}</Header>
+          <Form onSubmit={handleLogin} error={!!error}>
+            <Message error content={error} />
+            <Form.Input
+              autoFocus
+              maxLength="24"
+              placeholder={t("register.form.nickname")}
+              name={t("register.form.nickname")}
+              value={nickname}
+              onChange={(_, { value }) => setNickname(value)}
+            />
+            <Form.Group>
+              <Form.Button content={t("login.actions.login")} />
+            </Form.Group>
```

Comment:
There's no way to change login method once you click on "Guest"

Reply:
good point fixing it

Reply:
fixed